doc/nrf/device_guides/nrf52/testing.rst

```diff
@@ -0,0 +1,36 @@
+.. _nrf52_computer_testing:
+
+Testing with multiple nRF52 Series devices
+##########################################
+
+If you have an nRF52 Series DK with the :ref:`peripheral_uart` sample and either a dongle or second Nordic Semiconductor development kit that supports Bluetooth Low Energy, you can test the sample on your computer.
+Use the Bluetooth Low Energy app in `nRF Connect for Desktop`_ for testing.
+
+To perform the test, :ref:`connect to the nRF52 Series DK <nrf52_gs_connecting>` and complete the following steps:
```

Comment:
If this page is kept, it should rather point to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/test_and_optimize.html and not the getting started guide.